packages/mwp-api-proxy-plugin/src/handler.js

```diff
@@ -27,9 +27,12 @@ export function parseRequestQueries(request) {
  * The default API proxy handler - just calls the decorated `proxyApi$` method
  * and serializes the API responses
  */
-export default (request, reply) => {
-	request.proxyApi$(parseRequestQueries(request)).subscribe(
-		responses => reply(JSON.stringify({ responses })).type('application/json'),
-		err => reply(err) // 500 error - will only be thrown on bad implementation
+export default (request, h) => {
+	const handleResponses = responses =>
+		h.response(JSON.stringify({ responses })).type('application/json');
+
+	return request.proxyApi$(parseRequestQueries(request)).subscribe(
```

Comment:
this is the code that is causing the problem with the unit test - you are returning an Observable 'Subscriber' here, which I think Hapi doesn't know what to do with.

Instead, the observable needs to be turned into a promise before returning